content/posts/2023/pdf-root-files-preview.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Introducing PDF and ROOT file previews in REANA web"
+date: 2023-06-26T14:00:00+01:00
+---
+
+We are excited to announce that the REANA web interface now supports previewing of PDF and ROOT files.
+This means that instead of downloading these files to view them, you can simply inspect them right away,
+without leaving your browser!
+
+<!--more-->
+
+For a long time, the REANA web UI has allowed users to preview certain types of files present in the
+workflow workspace. This has proven to be very handy for users who want to quickly glance through
+images and text files in their workspaces without having to download them. We are now extending
+this capability to PDF and ROOT files as well.
+
+## PDF files preview
+
+Starting from REANA 0.9.1, when you click on a PDF file in your workflow's workspace in the web UI, an embedded
+PDF viewer will load the document. You will be able to scroll through pages, zoom in and out,
+or even download the PDF if necessary.
+You can use it to quickly inspect the content of any PDF generated by your analysis or uploaded to your
+workspace.
+Here is a sneak peek of how it looks:
+
+![PDF file preview](/images/pdf-files-preview.png)
+
+## ROOT files preview
+
+ROOT files can also be previewed directly in the web UI.
```

Comment:
It may be useful to link, since some REANA readers from other scientific disciplines may not be familiar with ROOT. Hence:

[ROOT](https://root.cern.ch/) files...